oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/binary/fixtures/datastore/AzureDataStoreFixture.java

```diff
@@ -94,12 +97,22 @@ public DataStore createDataStore() {
 
         String connectionString = Utils.getConnectionStringFromProperties(azProps);
         try {
-            CloudBlobContainer container = Utils.getBlobContainer(connectionString, containerName);
-            container.createIfNotExists();
+            boolean useSDK12 = Boolean.parseBoolean(azProps.getProperty(AZURE_SDK_12_ENABLED, "false"));
```

Comment:
What about tests/implementations dependent upon the AzureDataStoreFixture, we should have parametrized tests or some other mechanism that test those with both v12 & v8 versions. 
There are many command supproted in oak run utility which would have corresponding tests which should support above.